plugins/follow_me/follow_me.h

```diff
@@ -0,0 +1,194 @@
+#pragma once
+
+#include <iostream>
+#include <cmath>
+
+
+namespace dronecore {
+
+class FollowMeImpl;
+
+class FollowMe
+{
+public:
+    explicit FollowMe(FollowMeImpl *impl);
+    ~FollowMe();
+
+    /**
+     * @brief Results of FollowMe operations
+     */
+    enum class Result {
+        SUCCESS = 0, /**< @brief %Request succeeded. */
+        NO_DEVICE, /**< @brief No device connected. */
+        CONNECTION_ERROR, /**< @brief %Connection error. */
+        BUSY, /**< @brief Vehicle busy. */
+        COMMAND_DENIED, /**< @brief Command denied. */
+        TIMEOUT, /**< @brief %Request timeout. */
+        UNKNOWN /**< @brief Unknown error. */
+    };
+
+    /**
+     * @brief FollowMe configuration.
+     * Reference: https://docs.px4.io/en/advanced_config/parameter_reference.html#follow-target
+     */
+    class Configuration
+    {
+    public:
+        /**
+         * @brief Side to follow target from
+         */
+        enum class FollowDirection {
+            NONE = -1,
```

Comment:
I suggest to use enums without values like the Other APIs.

Reply:
ok.